app/services/phone_number_capabilities.rb

```diff
@@ -0,0 +1,32 @@
+class PhoneNumberCapabilities
+  VOICE_UNSUPPORTED_US_AREA_CODES = {
+    '648' => 'American Samoa',
+    '671' => 'Guam',
+    '670' => 'Northern Mariana Islands',
+    '340' => 'United States Virgin Islands',
+  }.freeze
+
+  attr_reader :phone
+
+  def initialize(phone)
+    @phone = phone
+  end
+
+  def sms?
+    true
+  end
+
+  def voice?
```

Comment:
so it looks like in all the uses of this, we do `!capabilities.voice?` -- maybe should we just invert it and switch to `capabilities.sms_only?`